src/cpp/benders/logger/MathLogger.cpp

```diff
@@ -0,0 +1,98 @@
+#include "logger/MathLogger.h"
+
+#include <iomanip>
+#include <sstream>
+
+void MathLoggerBase::setHeadersList() {
+  MathLogger::setHeadersList({ITE, LB, UB, BESTUB, ABSOLUTE_GAP, RELATIVE_GAP,
+                              MINSIMPLEX, MAXSIMPLEX, TIMEMASTER,
+                              SUB_PROBLEMS_TIME_CPU, SUB_PROBLEMS_TIME_WALL});
+}
+
+void MathLogger::setHeadersList(const std::list<std::string>& headers) {
+  headers_.clear();
+  headers_ = headers;
+}
+
+void MathLogger::write_header() {
+  setHeadersList();
+  for (const auto& header : Headers()) {
+    TheLogDestination() << header;
```

Comment:
`LogDestination()` instead of `TheLogDestination()` seems more natural

Reply:
i've a struct named LogDestination, i suggest LogsDestination